### ISSUE TITLE
Confusing documentation of Schedulers#boundedElastic regarding cached instance

### ISSUE THREAD

**Issue by dedeibel** [reactor/reactor-core#2989]
## Documentation Issue

I think the documentation of `Schedulers#boundedElastic` is confusing regarding the `@return` statement. It states

	 * @return a new {@link Scheduler} that dynamically create workers with an upper bound to
	 * the number of backing threads and after that on the number of enqueued tasks,
	 * that reuses threads and evict idle ones

It lead me to think that the returned **new instance** has to be stored and reused as a user. But looking into the code I noticed that there will only be one instance and that it will be cached.

The return comment seems more fitting for the method `newBoundedElastic`.

See also https://github.com/reactor/reactor-core/blob/748fc2b93b2e9c8534effe66a1f2962132d71cfd/reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java#L200

Minor remark: Additionally I think it should be: `@return a new {@link Scheduler} that dynamically create**s** workers …`

## Improvement Suggestion

It would suggest to clearly state that there will only be one instance created and that instance is returned on subsequent calls.

e.g.

	 * @return default instance of a {@link Scheduler} that dynamically creates workers with an upper bound to
	 * the number of backing threads and after that on the number of enqueued tasks,
	 * that reuses threads and evict idle ones. Only one instance of the Scheduler will be created on the first call
	 * and cached. The same instance is returned on subsequent calls

The methods `#single` and `#parallel` behave similarly and use this "default instance" term: `@return default instance of a Scheduler…`

**Comment by simonbasle** [comment]
yeah this looks like a copy-paste mistake, especially given the `parallel()` and `single()` method have more logical wording.
the `creates` remark is valid too, and this should be fixed in `newBoundedElastic` methods too.

@dedeibel do you want to submit a PR?

**Comment by dedeibel** [comment]
Can do.
